BackBLAST.py

```diff
@@ -281,8 +282,11 @@ def main(args):
                              '''The smaller this number is, the stricter the BLAST search.''')
 
     parser.add_argument('-i', '--min_ident', metavar='IDENT', default=DEFAULT_MINIMUM_IDENTITY_CUTOFF,
-                        help='''The minimum sequence identify cutoff for removing high scoring pairs. ''' +
+                        help='''The minimum sequence identify cutoff for removing high scoring pairs.''' +
                              '''The larger this number is, the stricter the BLAST search.''')
 
+    parser.add_argument('-o', '--output_file', metavar='OUTPUT', required=True,
+                        help='''Output CSV-format BLAST results table.''')
```

Comment:
Replace with something like "The path to the output file containing CSV-formatted BLAST results." Table is redundant. Indicate to the users that they are supposed to pass a file path.